x/staking/types/msg.go

```diff
@@ -209,8 +209,8 @@ func (msg MsgEditValidator) ValidateBasic() sdk.Error {
 
 // MsgDelegate - struct for bonding transactions
 type MsgDelegate struct {
-	DelegatorAddr sdk.AccAddress `json:"delegator_addr"`
-	ValidatorAddr sdk.ValAddress `json:"validator_addr"`
+	DelegatorAddr sdk.AccAddress `json:"delegator_address"`
+	ValidatorAddr sdk.ValAddress `json:"validator_address"`
```

Comment:
I'm a fan on consistency between the json and field name, so the change is fine, but I'd recommend changing the struct field name too aka:
```
ValidatorAddress sdk.ValAddress `json:"validator_address"`
```

Reply:
I am consistency addict too, though I'd prefer the shorter `delegator_addr` (fewer bytes in JSON objects).
However if we change this here, please remember do update `x/distribution` files too:

```
alessio@bangalter:~/.../cosmos/cosmos-sdk$ grep -rw -l validator_addr x/
x/staking/types/msg.go
x/staking/types/delegation.go
x/staking/client/rest/tx.go
x/distribution/types/msg.go
x/distribution/types/genesis.go
x/distribution/keeper/querier.go
```

Reply:
we can't and shouldn't use `delegator_addr` @alessio, but I agree I missed a bunch of instances. Thank you!

Reply:
Updated.